end-user-application-machines/new-agent-vm/Install-PCoIPAgent.ps1

```diff
@@ -18,6 +18,9 @@ Configuration InstallPCoIPAgent
         [String]$sumoCollectorID,
 
         [Parameter(Mandatory=$false)]
+        [String]$sasToken,
```

Comment:
Should we make this a credential populated by a SecureString in the JSON like we're handling other sensitive data?

Reply:
I'll look into how to do that.  i don't think we are passing this anywhere else to the DSC. 

We pass the registration code as PSCredential but in this case i don't know where to cast it as pscredential